File: SDKs/Aspose.Words_Cloud_SDK_for_CSharp/src/Com/Aspose/Words/Model/WatermarkText.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Words.Model {
  public class WatermarkText {
    public string Text { get; set; }

    public double RotationAngle { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class WatermarkText {\n");
      sb.Append("  Text: ").Append(Text).Append("\n");
      sb.Append("  RotationAngle: ").Append(RotationAngle).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: Examples/Android/Aspose-Words-Cloud-SDK-for-Android/src/main/java/com/aspose/words/model/WatermarkText.java
package com.aspose.words.model;

public class WatermarkText {
  private String Text = null;
  private Double RotationAngle = null;
  /**
	 * getText
	 * Gets String
	 * @return Text
	 */
  public String getText() {
    return Text;
  }

	/**
	 * setText
	 * Sets String
	 * @param Text String
	 */
  public void setText(String Text) {
    this.Text = Text;
  }

  /**
	 * getRotationAngle
	 * Gets Double
	 * @return RotationAngle
	 */
  public Double getRotationAngle() {
    return RotationAngle;
  }

	/**
	 * setRotationAngle
	 * Sets Double
	 * @param RotationAngle Double
	 */
  public void setRotationAngle(Double RotationAngle) {
    this.RotationAngle = RotationAngle;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class WatermarkText {\n");
    sb.append("  Text: ").append(Text).append("\n");
    sb.append("  RotationAngle: ").append(RotationAngle).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

